keras/layers/reshaping/cropping2d.py

```diff
@@ -57,9 +57,10 @@ class Cropping2D(Layer):
         `(batch_size, height, width, channels)` while `channels_first`
         corresponds to inputs with shape
         `(batch_size, channels, height, width)`.
-        It defaults to the `image_data_format` value found in your
-        Keras config file at `~/.keras/keras.json`.
-        If you never set it, then it will be "channels_last".
+        When unspecified, uses
+        `image_data_format` value found in your Keras config file at
+         `~/.keras/keras.json` (if exists) else 'channels_last'.
+        Defaults to 'channels_last'.
```

Comment:
Please change the quote from ' to ` for consistency. Same for all the changes in this PR.

Reply:
`'` refers to string whereas ``` refers to some sort of keyword.